app/code/community/Vindi/Subscription/Helper/Api.php

```diff
@@ -473,11 +473,13 @@ public function approveBill($billId)
     }
 
     /**
-     * @param $billId
+     * Cancela a assinatura e fatura na Vindi após falha no pagamento
+     *
+     * @param $vindiId
      */
-    public function deleteBill($billId)
+    public function deletePurchase($vindiId, $type)
```

Comment:
Não é melhor deixar um nome mais semântico?
```suggestion
    public function cancelSubscription($vindiId, $type)
```

Reply:
Aqui ele cancela tanto uma subscription (e suas faturas), ou faturas avulsas.
Pensei em separar em métodos, porém, a lógica teria de ser inserida no fluxo de pagamento de um pedido.
Acho os métodos muito semelhantes para essa abstração :thinking: 
Podemos trabalhar isso?

Reply:
```suggestion
    public function cancelPurchase($vindiId, $type)
```